commands/api.go

```diff
@@ -93,6 +93,22 @@ func isValidRelpath(relpath string) (error, bool) {
 	return nil, true
 }
 
+func hasPathParameters(path string) bool {
+	return (strings.Count(path, "{") > 0) || (strings.Count(path, "}") > 0)
```

Comment:
Is `path` just JSON?

Reply:
Nope, the path is something like `"/users/{user}/balance"`, so we're checking for balanced brackets.  But now that you bring it up, I think Mark and I had discussed changing this strategy up a bit so that the logic uses regex instead.